DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc

```diff
@@ -266,13 +266,13 @@ void TrackBuildingAnalyzer::initHisto(DQMStore::IBooker & ibooker, const edm::Pa
     numberOfTrajCandsPerSeedVsEta->setAxisTitle("Seed #eta", 1);
     numberOfTrajCandsPerSeedVsEta->setAxisTitle("Number of Trajectory Candidates for each Seed", 2);
 
-    histname = "SeedStoppingSourceVsNumberOfTrajCandsPerSeed_"+seedProducer.label() + "_";
-    seedStoppingSourceVsNumberOfTrajCandsPerSeed = ibooker.book2D(histname+CatagoryName, histname+CatagoryName, candsBin, candsMin, candsMax, stopReasonSize, 0, stopReasonSize);
+    histname = "StoppingSourceVsNumberOfTrajCandsPerSeed_"+seedProducer.label() + "_";
```

Comment:
I'd suggest to keep the "Seed" prefix to highlight the difference wrt. the "track stopping source histograms" and instead fix a likely bug caused by #20450 by renaming
* line 244: "StoppingSourceVsPhi_" -> "SeedStoppingSourceVsPhi_"
* line 249: "StoppingSourceVsEta_" -> "SeedStoppingSourceVsEta_"

(or, at least the histogram naming bug should be fixed, I don't really mind if this histogram starts with "Seed" or not as long as it is clear that it is the "seed stopping reason")

Reply:
ok, I will have a look, I was trying to keep histogram together as they are plotted in alphabetic order...

Reply:
Ok, maybe move the "Seed" to some other place in the name or something then?